### DISCUSSION TITLE
Offline Edit

### DISCUSSION THREAD

**Discussion by Gomilu** [GitbookIO/community#24] [Feature Requests]
### What's your feature/idea?

Please describe here - in as much detail as you can - your idea.

Ability to edit one's gitbook collections, spaces, pages offline and simply sync when coming online

### What's your desired outcome?

What would you like to be able to do, if your idea came to life?

Benefit of being able to work/study anywhere, even without connection

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

**Comment by piranna** [comment]
This seems to be related to https://github.com/GitbookIO/community/discussions/7
